include/NovelRT.Interop/Ecs/NrtEcs.h

```diff
@@ -0,0 +1,7 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
```

Comment:
(I cant drag for multilines on mobile x.x)
Is this file necessary?

Reply:
Yes, its the namespace file, I've just not filled it in yet with the relevant includes.

Reply:
Okayyyy just making sure it didn't slip or anything! 😄